lib/graphql/generators/loader_generator.rb

```diff
@@ -0,0 +1,15 @@
+require 'graphql/generators/base_generator'
+module GraphQL
+  module Generators
+    # Generate a batch loader by name.
+    #
+    # ```
+    # rails g graphql:loader RecordLoader
+    # ```
+    #
+    # Should we have a loader for the
+    # nearly-universal foreign key loader?
```

Comment:
> Should we have a loader for the nearly-universal foreign key loader?

:-1: Personally I think any code that's "nearly-universal" should probably be in a gem rather than generated into the application codebase. Not sure whether `graphql-ruby` is the most appropriate gem or not--but maybe `graphql-activerecord`?